khal/ui/__init__.py

```diff
@@ -1379,11 +1379,20 @@ def start_pane(pane, callback, program_info='', quit_keys=['q']):
         quit_keys=quit_keys,
     )
     frame.open(pane, callback)
-    palette = _add_calendar_colors(getattr(colors, pane._conf['view']['theme']),
-                                   pane.collection)
-    loop = urwid.MainLoop(frame, palette,
-                          unhandled_input=frame.on_key_press,
-                          pop_ups=True)
+    palette = _add_calendar_colors(
+        getattr(colors, pane._conf['view']['theme']), pane.collection)
+    loop = urwid.MainLoop(
+        frame, palette, unhandled_input=frame.on_key_press, pop_ups=True)
+    frame.loop = loop
+
+    def redraw_today(loop, pane, round=[0]):
+        round[0] += 1
+        frame.alert('redraw {}!'.format(round[0]))
```

Comment:
This is totally a proof of concept, we also really don't want to query the database every 60 seconds.